### ISSUE TITLE
[FloatingActionButton] `minTouchTargetSize` doesn't work

### ISSUE THREAD

**Issue by pubiqq** [material-components/material-components-android#3256]
[14-235-1.webm](https://user-images.githubusercontent.com/82187521/220647281-9aaa9787-dc93-4e5f-9cfd-aabdb3167f9b.webm)

<br>
<b>Material Library version:</b> 1.9.0-alpha02

**Comment by drchen** [comment]
Oh my..... thanks for noticing this. 

I did some quick check and found two issues:

1. FABs are using a hard-coded dimension value instead of the global theme attribute. (or even a styleable attribute)
2. More seriously, the FAB implementation of min touch target size doesn't work - it only increases the area FAB occupies but the extra area is not clickable. 

This sounds like a huge problem to me and I'll try to make this prioritized.

**Comment by drchen** [comment]
It's fixed by https://github.com/material-components/material-components-android/commit/d6f36e89e2f815a4d72884618b62e946aaf7b6d9
